immix/src/allocator/global_allocator.rs

```diff
@@ -21,6 +21,10 @@ pub struct GlobalAllocator {
     heap_end: *mut u8,
     /// 所有被归还的Block都会被放到这个Vec里面
     free_blocks: Vec<(*mut Block, bool)>,
+    /// 所有被归还的Block都会被放到这个Vec里面
+    unavailable_blocks: Vec<*mut Block>,
+    /// 所有被归还的Block都会被放到这个Vec里面
+    recycle_blocks: Vec<*mut Block>,
```

Comment:
what's the difference between `unavailable_blocks` and `recycle_blocks`? The comments are same.